### DISCUSSION TITLE
liquidctl-git installation error and liquidctl NZXT Kraken 2023 Elite (experimental) not detected

### DISCUSSION THREAD

**Discussion by ghost** [liquidctl/liquidctl#684] [Q&A] [answered]
Hi, I'm having some difficulty getting my AiO going. I managed it a while back, around October, with liquidctl-git, but now when I attempt to install it I get an error. I'm able to install regular liquidctl, though the device isn't detected.

Here's what that looked like before. I'm still using all the same old hardware, though I imagine the version of Arch that I'm using is newer.
```
Device #0: NZXT Kraken 2023 Elite (experimental)
├── Vendor ID: 0x1e71
├── Product ID: 0x300c
├── Release number: 0x0200
├── Serial number: 473892820E48211B
├── Bus: hid
├── Address: /dev/hidraw8
└── Driver: KrakenZ3
```

[liquidctl-git installation log](https://pastebin.com/ZJicefii)

I suspect the installation issue is to do with the rule file. I can download the project from github and modify the file, but I don't know how to then install liquidctl-git from the downloaded project.

**Comment by aleksamagicka** [comment, marked as answer]
Seems similar to #618 and https://github.com/liquidctl/liquidctl/issues/645#issuecomment-1834150649. Have you perhaps tried manually setting the permissions file as described in [Allowing access to the devices](https://github.com/liquidctl/liquidctl/#allowing-access-to-the-devices) section from the README?

**Reply by ghost** [reply]
Yeah, I was under the impression the package name became liquidctl-git, and didn't stay as liquidctl. My mistake.

I was able to install liquidctl and my deivce is recognised.

[lojith@overlord ~]$ liquidctl status
NZXT Kraken 2023 Elite (broken)
├── Liquid temperature    28.4  °C
├── Pump speed            1145  rpm
├── Pump duty               25  %
├── Fan speed                0  rpm
└── Fan duty                20  %
